### PR TITLE
Deprecate some long overdue functions

### PR DESCRIPTION
**Pull request by AlexejPenner**
## Describe changes
I removed some deprecated functionality

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

**Review by htahir1 — APPROVED** [review]

**Comment by htahir1** [comment]
Sorry it wasnt an approval yet! :D ! Also @stefannica can comment on the secret managers stuff

**Review by fa9r — COMMENTED** [review]
How come workspace CRUD is deprecated?

**Review by fa9r — APPROVED** [review]

File: src/zenml/cli/utils.py [hunk 1]
```diff
@@ -1554,6 +1554,18 @@ def warn_deprecated_secrets_manager() -> None:
         "documentation page for more information."
     )
 
+def fail_secrets_manager_creation() -> None:
+    """Warning for deprecating secrets managers."""
+    error(
+        "Creating secrets managers is longer supported. They will be "
```
[the remark this answers is not in the document]
Comment (AlexejPenner) [reply src/zenml/cli/utils.py]: I rephrased, lemme know if it is better

**Review by htahir1 — APPROVED** [review]
Thank you for doing it , i hope @stefannica  knows we are throwing an error now though

**Review by stefannica — APPROVED** [review]

**Comment by stefannica** [comment]
> Thank you for doing it , i hope @stefannica knows we are throwing an error now though

A much needed measure !

**Pull request merged**